### ISSUE TITLE
why some videos can't be reproduced, some days after installation?

### ISSUE THREAD

**Issue by hunkjazz** [iv-org/invidious#1309]
It's always famous songs.

It seems to be something related to copyright.

Since [Invidious][1] main instance don't ever face this issue, I was wondering if you could tell me what's the solution.

_I'm running Invidious on Fedora 32, using `localhost:3000`._

[1]: https://invidio.us

**Comment by TheFrenchGhosty** [comment]
Why do you mean by "can't be reproduced"? Are you sure you're not blocked by Google?

**Comment by hunkjazz** [comment]
I don't know if this is a bug.
I'm just a newbie.

This is the message output:

> The media could not be loaded, either because the server or network failed or because the format is not supported.

**Comment by hunkjazz** [comment]
I made videos work again by doing this:

1. `sudo service invidious stop`;
2. `cd /home/invidious/invidious`;
3. `./invidious`;
4. visited video and it finally worked;
5. hit `ctrl+c` to stop process;
6. `sudo service invidious start`;
7. visited video and it was still working.

Some considerations:

- I have no idea is this is Google blocking me;
  - would you mind referencing some links, so I can learn about it?
- this issue has nothing to do with #92;
- this issue is not solved with `quality=dash`;
- I don't know if I solved this issue because I restarted the service or because I ran `./invidious`.

**Comment by reivilibre** [comment]
I also seem to get this issue occasionally; the video frame blinks with some error saying it could be due to the format or that the server failed (I just restarted so can't exactly reproduce the error now, sorry).

I notice in the developer console that I am getting 403s from (????).googlevideo.com and warnings about Content Security Policy blocking script-src or something… I'll update here when I get the precise messages again.

**Comment by reivilibre** [comment]
An excerpt of the browser's console when this is going on:

```
An error occured in the player, reloading... player.js:65:21
GEThttps://r1---sn-hoxu-h0js.googlevideo.com/videoplayback?expire=1605619763&ei=03uzX8-5MdOA-gb1oZ6oAw&ip=79.143.178.141&id=o-AI0nLVEMtuEm1Q8XCppc_CFmFb0OxGLR2Mu2JdyDTy70&itag=18&source=youtube&requiressl=yes&mh=En&mm=31%2C29&mn=sn-hoxu-h0js%2Csn-4g5ednsd&ms=au%2Crdu&mv=u&mvi=1&pl=26&vprv=1&xtags=tx%3D23958302&mime=video%2Fmp4&ns=T3X2D4xHpUvA0WM1_iJyRgEF&gir=yes&clen=106197162&ratebypass=yes&dur=3837.051&lmt=1604357387754076&mt=1605597922&fvip=1&c=WEB&txp=1116222&n=Gt-LXSuOuphouXP&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Cxtags%2Cmime%2Cns%2Cgir%2Cclen%2Cratebypass%2Cdur%2Clmt&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl&lsig=AG3C_xAwRgIhAK1BKbjXXiTpQbIGcvv8ai-sIHhE5tlMiu0nrXcokMJgAiEA_2DXiOHqgo1ouj0881ROzv8Jmtv0rBk3vAf09BXUlyA%3D&sig=YUOLqW=6Eyh4LfEN6v3a7NJEFqQPBn_HG_8gsneuAEiAbylb7_RWr_9HL7LdcMlEeiX3cxyMxNqRwSFsdzavAOLAGIgRw8JQ0qOG&host=r1---sn-hoxu-h0js.googlevideo.com
[HTTP/1.1 403 Forbidden 99ms]

HTTP load failed with status 403. Load of media resource https://MYINSTANCE/latest_version?id=_pLO4jFDeIc&itag=18 failed. watch
VIDEOJS: ERROR: (CODE:4 MEDIA_ERR_SRC_NOT_SUPPORTED) The media could not be loaded, either because the server or network failed or because the format is not supported. 
Object { code: 4, message: "The media could not be loaded, either because the server or network failed or because the format is not supported." }
video.min.js:12:695
Content Security Policy: The page's settings blocked the loading of a resource at eval ("script-src").
```


Some videos appear to work fine, but a few like these do not -- but as soon as I restart Invidious, the browser requests https://MYINSTANCE/latest_version?id=_pLO4jFDeIc&itag=18 once more and it now succeeds — the play button appears and I can watch the video.

**Comment by andrewzah** [comment]
I had the same issue. Couldn't play a video, got 403 errors in console. Then after restarting invidious I was able to play it.

**Comment by peperunas** [comment]
I agree, I have the same issue and it is not related with #92.
The workaround by @andrewzah, restarting Invidious, works but it is rather annoying to have to restart the instance every X hours to enable the reproduction of certain videos.

The error is always the same:

```
The media could not be loaded, either because the server or network failed or because the format is not supported.
```

**Comment by peperunas** [comment]
In addition, I think #1312 is related to this open issue. The bug seems to be the same.

**Comment by lynchpatrickj** [comment]
Having the same issue. Restarting invidious.service seems to make the videos playable again but no idea what the cause is.

**Comment by andrigamerita** [comment]
Some considerations I got out of personal tests.

The issue seems to be happening only on "YouTube-approved" music videos, meaning videos of musical genre
- either uploaded by YouTube users with a music author partnership with YouTube
- or part of a so-called "Topic" channel, which is a music library automatically generated by YouTube
It doesn't seem to appear on any video that has "music in it" (stated by the official YouTube website in the video description), but just those kinds that I mentioned, confirming what @hunkjazz originally thought (`It's always famous songs.`)

The issue is not related to georestrictions that the video might have, as already mentioned by some users in this thread.
The issue manifests itself regardless of the preferred video quality set in the user preferences, so it's not an issue with DASH or anything of that matter.
The issue seems to present itself only after a day or two that the Invidious instance is running. Killing and restarting the instance fixes the problem, until it inevitably reappears some time later, which was also confirmed by some users in this thread.

**Comment by unixfox** [comment]
Can't reproduce the issue if proxy is enabled, the video loads fine.
Closing in favor of https://github.com/iv-org/invidious/issues/2850 for forcing proxying.
